File: src/models/BoxMeasurement.ts
export interface BoxMeasurement {
  comprimento: number;  // C em cm
  largura: number;      // L em cm
  altura: number;       // A em cm
}

export function volumeM3(m: BoxMeasurement): number {
  return (m.comprimento * m.largura * m.altura) / 1_000_000;
}

export function pesoCubado(m: BoxMeasurement): number {
  return (m.comprimento * m.largura * m.altura) / 6_000;
}

export function displayText(m: BoxMeasurement): string {
  return `C: ${Math.round(m.comprimento)} × L: ${Math.round(m.largura)} × A: ${Math.round(m.altura)} cm`;
}

File: MedidorCaixas/Models/BoxMeasurement.swift
import Foundation

struct BoxMeasurement {
    var comprimento: Double  // C - cm
    var largura: Double      // L - cm
    var altura: Double       // A - cm

    var volumeM3: Double { comprimento * largura * altura / 1_000_000 }
    var pesoCubado: Double { comprimento * largura * altura / 6_000 }

    var displayText: String {
        "C: \(Int(comprimento.rounded())) × L: \(Int(largura.rounded())) × A: \(Int(altura.rounded())) cm"
    }
}
